Nishant/postgres assignment/src/services/class.ts

```diff
@@ -0,0 +1,29 @@
+import sqlclient from '../database/db';
+
+export async function getClassService() {
+  try {
+    await sqlclient.query('SET search_path TO College');
+    return (await sqlclient.query('SELECT * FROM Class_schedule ORDER BY classno'));
+  } catch (e) {
+    throw Error(e);
+  }
+}
+
+export async function getStudentByClassIdService(classId:number) {
+  try {
+    await sqlclient.query('SET search_path TO College');
```

Comment:
search_path can be replaced. ref : https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment or feel free to get in touch with jatin/tarun...